scripts/updateLicense.py

```diff
@@ -79,7 +95,21 @@ def main():
     for name in sys.argv[1:]:
         if name.endswith('.go'):
             try:
-                update_go_license(name)
+                update_license(name, LICENSE_BLOB_LINES_GO)
+            except Exception as error:
+                logger.error('Failed to process file %s', name)
+                logger.exception(error)
+                raise error
+        elif name.endswith('.sh'):
+            try:
+                update_license(name, LICENSE_BLOB_LINES_SHELL_AND_MAKEFILE)
+            except Exception as error:
+                logger.error('Failed to process file %s', name)
+                logger.exception(error)
+                raise error
+        elif 'Makefile' in name:
+            try:
+                update_license(name, LICENSE_BLOB_LINES_SHELL_AND_MAKEFILE)
```

Comment:
Reminder: the blocks inside `if` and `elif`'s are somewhat repeated.

Reply:
You only need to determine comment prefix based on file type. Then try/catch can be done once.